src/c++/CMakeLists.txt

```diff
@@ -3,14 +3,14 @@
 # Redistribution and use in source and binary forms, with or without
 # modification, are permitted provided that the following conditions
 # are met:
-#  * Redistributions of source code must retain the above copyright
-#    notice, this list of conditions and the following disclaimer.
-#  * Redistributions in binary form must reproduce the above copyright
-#    notice, this list of conditions and the following disclaimer in the
-#    documentation and/or other materials provided with the distribution.
-#  * Neither the name of NVIDIA CORPORATION nor the names of its
-#    contributors may be used to endorse or promote products derived
-#    from this software without specific prior written permission.
+# * Redistributions of source code must retain the above copyright
+# notice, this list of conditions and the following disclaimer.
+# * Redistributions in binary form must reproduce the above copyright
+# notice, this list of conditions and the following disclaimer in the
+# documentation and/or other materials provided with the distribution.
+# * Neither the name of NVIDIA CORPORATION nor the names of its
+# contributors may be used to endorse or promote products derived
+# from this software without specific prior written permission.
```

Comment:
clang-format 6.0 is the agreed upon formatter for C++ code, not CMake code. I'm not sure if we have an agreed-upon formatter for CMake code.

Reply:
To avoid `clang-format` auto-formatting non-C++ files, I have this in my vscode `settings.json`:
```
{
    ...
    "[cpp]": {
        "editor.defaultFormatter": "xaver.clang-format",
    },
    ...
}
```

Reply:
Updated my settings.json file and fixed spacing.